mobx_codegen/lib/src/template/params.dart

```diff
@@ -3,17 +3,16 @@ class ParamTemplate {
   String type;
   String defaultValue;
   bool hasRequiredAnnotation = false;
+  bool hasRequiredKeyword = false;
 
   String get asArgument => name;
 
   NamedArgTemplate get asNamedArgument => NamedArgTemplate()..name = name;
 
-  String get metadata => hasRequiredAnnotation ? '@required ' : '';
+  String get metadata => (hasRequiredAnnotation ? '@required ' : '') + (hasRequiredKeyword ? 'required ' : '');
```

Comment:
I think the annotation is no longer valid in this version of Dart, let's just use the keyword check instead

Reply:
ok